### PR TITLE
Fix map_target documentation

### PR DESCRIPTION
**Pull request by svenwest**

### CODE REVIEW

File: lib/1.4/utility.dml [hunk 1]
```diff
@@ -1268,7 +1270,8 @@ template map_target is (connect, _qname) {
        * `read(uint64 addr, uint64 size) -> (uint64) throws`
 
          Reads `size` bytes starting at `addr` in the connected object. Size
-         must be 8 or less. Throws an exception if the read fails.
+         must be 8 or less. Byte-order is little-endian. Throws an exception if
```
Comment (mandolaerik) [inline lib/1.4/utility.dml:1273]: I think it's "byte order" rather than "byte-order", but I'm not sure.

**Review by mandolaerik — APPROVED** [review]
OK if your research shows that "byte-order" is a correct form.

**Review by mandolaerik — APPROVED** [review]

**Comment by syssimics** [comment]
TeamCity build completed (SUCCESS): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9707480

**Comment by syssimics** [comment]
TeamCity build completed (SUCCESS): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9707479

**Comment by syssimics** [comment]
TeamCity build completed (SUCCESS): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9710857

**Comment by syssimics** [comment]
TeamCity build completed (SUCCESS): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9710865

**Comment by syssimics** [comment]
TeamCity build completed (SUCCESS): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9710872

**Comment by syssimics** [comment]
TeamCity build completed (FAILURE): https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=9710871

**Pull request merged**